tests/modal.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { test } from "@playwright/test";
```

Comment:
naming (blocking): The name of the test is too generic. Consider calling it something like`leave-room.spec.ts` which is more in line with the names of the other tests.